cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -808,8 +809,13 @@ def _add_cfn_volume(self, id: str, shared_ebs: SharedEbs):
 
     def _add_wait_condition(self):
         wait_condition_handle = cfn.CfnWaitConditionHandle(self, id="HeadNodeWaitConditionHandle" + self.timestamp)
+        head_node_bootstrap_timeout = get_attr(self.config, "dev_settings.timeouts.head_node_bootstrap_timeout")
         wait_condition = cfn.CfnWaitCondition(
-            self, id="HeadNodeWaitCondition" + self.timestamp, count=1, handle=wait_condition_handle.ref, timeout="1800"
+            self,
+            id="HeadNodeWaitCondition" + self.timestamp,
+            count=1,
+            handle=wait_condition_handle.ref,
+            timeout=str(head_node_bootstrap_timeout) if head_node_bootstrap_timeout else str(NODE_BOOTSTRAP_TIMEOUT),
```

Comment:
oh, saw now that the default is set here. You can move the default in the get_attr and leave this cleaner as `timeout=head_node_bootstrap_timeout`

Reply:
Good idea, changed for all places